src/primitives/orchard.h

```diff
@@ -108,6 +113,18 @@ class OrchardBundle
             return std::nullopt;
         }
     }
+
+    bool OutputsEnabled() const {
+        return orchard_bundle_outputs_enabled(inner.get());
+    }
+
+    bool SpendsEnabled() const {
+        return orchard_bundle_spends_enabled(inner.get());
+    }
+
+    bool HasValidCoinbaseOutputs() const {
```

Comment:
Consider renaming this to `CoinbaseOutputsAreValid` (to avoid the implication that there must be any such outputs).